src/lib/util.js

```diff
@@ -9,23 +9,28 @@ const car = "car/";
 const user = "user/";
 const member = "member/";
 const auth = "auth/";
-
+const personal = "personal";
+const plan = "plan/";
 export const urls = {
     get: {
-        memberPersonalData: baseURL + member + "personal",
+        memberPersonalData: baseURL + member + personal,
         isLoggedIn: baseURL + user + "whoami",
-        reservationPersonalData: baseURL + reservation + "personal",
+        reservationPersonalData: baseURL + reservation + personal,
         singleCar: baseURL + car,
+        allPlans: baseURL + plan + "all",
     },
     post: {
         forgotPassword: baseURL + auth + "forgot-password",
         register: baseURL + auth + "register",
         login: baseURL + auth + "login",
         logout: baseURL + auth + "logout",
     },
-    put: {},
+    put: {
+        newPersonalData: baseURL + member + personal,
+        newPlanData: baseURL + plan + personal,
+    },
     delete: {
-        singleReservation: baseURL + reservation + "personal/",
+        singleReservation: baseURL + reservation + personal + "/",
```

Comment:
Warum hat die URL ein `/` am Ende und die (meisten) anderen nicht?

Reply:
weil in dem fetch zur Löschung später noch die id angefügt wird und dazwischen halt ein / stehen muss. Habe das an dieser stelle gemacht, weil ich das schöner fand es im fetch selber dazwischen zu packen